src/app/components/action-card-component/action-card-component.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { ActionCardContentInterface } from './action-card-content-interface';
+
+@Component({
+  selector: 'app-action-card-component',
+  templateUrl: './action-card-component.component.html',
+  styleUrls: ['./action-card-component.component.scss'],
+})
+export class ActionCardComponentComponent {
+  isListOpen: boolean;
```

Comment:
isListOpen: boolean;
showPopover: boolean;

these properties should go after all Inputs